ambry-store/src/test/java/com.github.ambry.store/LogSegmentTest.java

```diff
@@ -588,8 +684,13 @@ private void doAppendTest(Appender appender) throws IOException {
       }
 
       // read and ensure data matches
-      readAndEnsureMatch(segment, writeStartOffset, bufOne);
-      readAndEnsureMatch(segment, writeStartOffset + bufOne.length, bufTwo);
+      if (Platform.isLinux()) {
+        readDirectlyAndEnsureMatch(segment, writeStartOffset, bufOne);
+        readDirectlyAndEnsureMatch(segment, writeStartOffset + bufOne.length, bufTwo);
+      } else {
+        readAndEnsureMatch(segment, writeStartOffset, bufOne);
```

Comment:
this can be done anyway I think? i.e. don't put it as an `else`

Reply:
good point. Fixed.